### ISSUE TITLE
Is this repro still maintained?

### ISSUE THREAD

**Issue by CarliJoy** [bharling/django-pint#15]
Dear @bharling,

is this repro still maintained?
It wasn't updated in four years.

Would you be okay with a change of a maintainer?

Here is one documentation of how to do it:
https://medium.com/@shazow/how-to-hand-over-an-open-source-project-to-a-new-maintainer-db433aaf57e8

I am interested to continue working on this project, merge the open pull request, make it work with Python 3 and django>2.2.
But I would drop support for Python 2 and django below 2.2. as it deprecated by now.

What do you think?

**Comment by kc2684** [comment]
Dear @CarliJoy & @bharling 

I have got it working, locally with Python 3 and Django 3.1.1by removing 'context' argument from the 'fields.from_db_value' function.
I also added the 'unit_choices' argument to the model so these choices appear in Django's admin pages.
But i am very new to Git/Python & Django so bare with me while i figure out how to load these changes into my GitHub fork.

**Comment by CarliJoy** [comment]
@bharling I really would like to put some effort in this app as I need to use it for a later project. I am a fan continuity and collaboration. But as I need a working and updated version of the app for my project I would be forced to fork your app under a new name. IMO this always the worst solution for the community. Would be glad to get an answer :-)

@kc2684 yep also got it to work, see my fork of the repro.

**Comment by bharling** [comment]
Sorry - Yes I'm open to maintainers definitely!

**Comment by bharling** [comment]
@CarliJoy Yes if you're interested I'd welcome a change of maintainer, I don't have the spare time for this at all unfortunately - I'm flattered that so many people seem to have an interest in it :)

If you're up for the job it's yours ?

**Comment by CarliJoy** [comment]
Great :smiley: . Where can I sign :wink:  ?
You can find my mail address on my profile in case you need it.

So I guess will wait until you found the time to perform the transition :relaxed:

In the meantime I will try to think of some unit test to make coverall happy :clap:

**Comment by bharling** [comment]
Great - I've never transferred a repo before - I tried to transfer to you but it wouldn't let me because you have a fork of it? Would you be able to delete your fork and I'll give it another try?

thanks for taking this on!

**Comment by CarliJoy** [comment]
I deleted the fork (thank to git, there is always a local copy).
You can try again.

**Comment by bharling** [comment]
Done - would be grateful if you can add a CONTRIBUTORS list and pop me in there too 👍 - thanks again for taking this on

**Comment by CarliJoy** [comment]
Thanks - it worked :-)
You already have the AUTHORS list, is this not the same ?
Marked you as original a author there already. 
Or do you prefer a CONTRIBUTERS list? If yes should we keep the AUTHORS and what is the difference if yes?

Could you also add me as own in the PyPi Project:
https://github.com/pypa/warehouse/issues/1745

**Comment by CarliJoy** [comment]
@bharling  PS: My account  in pypi is [kound](https://pypi.org/user/kound/)

**Comment by CarliJoy** [comment]
@bharling I updated to package so far. So last step would be to tag it to publish to pypi.
So only your approval for the package owner or maintainer is needed, then everybody can enjoy the new version.

**Comment by cornelv** [comment]
@CarliJoy  I am sorry that this is off-topic, I am not sure where to ask this question..
I have some free time in the following weeks, let me know if I can help with any development work. 
thank you

**Comment by CarliJoy** [comment]
@cornelv: Added issues that are worth spending time one. Thanks for your offer :-)
